experiments/conductor/cmd/runner/runner.go

```diff
@@ -123,9 +123,13 @@ type Branch struct {
 	ProtoMsg  string `yaml:"proto-msg"`     // google.ai.generativelanguage.v1beta.Model
 	HostName  string `yaml:"host-name"`     // generativelanguage.googleapis.com
 
-	Notes []string `yaml:"notes"` // Observation goes here
+	Notes       []string `yaml:"notes"`        // Observation goes here
+	ApisEnabled []string `yaml:"apis-enabled"` // GCP APIs that need to be enabled
```

Comment:
when possible can we add a concrete example for typed data? Notes is more free form so its not a great example. The other fields have actual field values to make it more clear.

Reply:
done